pytorch_to_returnn/torch/nn/modules/normalization.py

```diff
@@ -53,6 +53,51 @@ def import_params_torch_to_returnn(self, *, layer: LayerNormLayer, torch_module:
     layer.params["bias"].load(torch_module.bias.detach().numpy(), session=session)
 
 
+class GroupNorm(Module):
+  def __init__(self, num_groups: int, num_channels: int, eps: float = 1e-5, affine: bool = True) -> None:
+    super(GroupNorm, self).__init__()
+    assert num_groups == 1 or num_groups == num_channels, "Not implemented otherwise"
+    self.num_groups = num_groups
+    self.num_channels = num_channels
+    self.eps = eps
+    self.affine = affine
+    if self.affine:
+        self.weight = Parameter(Tensor(num_channels))
+        self.bias = Parameter(Tensor(num_channels))
+    else:
+        self.register_parameter('weight', None)
+        self.register_parameter('bias', None)
+    self.reset_parameters()
+
+  def reset_parameters(self) -> None:
+      if self.affine:
+          init.ones_(self.weight)
+          init.zeros_(self.bias)
+
+  def extra_repr(self) -> str:
+      return '{num_groups}, {num_channels}, eps={eps}, affine={affine}'.format(**self.__dict__)
+
+  def import_params_torch_to_returnn(self, *, layer: NormLayer, torch_module: GroupNorm):
+    if isinstance(layer, SubnetworkLayer):
```

Comment:
I don't exactly understand, how can this happen?
This should never happen. The layer it gets here should be exactly the layer created via `create_returnn_layer_dict`. If this is not the case, sth is wrong.

Reply:
This happens in the case of `Fp32GroupNorm` in fairseq. Not yet exactly sure why. It is a subclass of `GroupNorm` and uses `torch.nn.functional.group_norm` in its `forward`. Should we handle that in a separate issue/PR?

Reply:
Ah, subclass? Can you put a link?
But anyway, that would be a separate issue. This should never be a `SubnetworkLayer`. It should be exactly be the layer you create in `create_returnn_layer_dict`.

Reply:
Ok, I will create a new issue for that.